.tools/readmes/README.md

```diff
@@ -20,10 +20,14 @@ and install packages by running the following commands in the
 `.tools/readmes` folder:
 
 ```
+cd .tools/readmes
```

Comment:
Can you update lines 16-19 to be more inline with these changes?

Reply:
Done